docs/UserGuide.md

```diff
@@ -10,65 +10,126 @@ This guide will help provide you with all the necessary information to set up, r
-    * [Canceling a meeting: `meet`](#canceling-a-meeting-meet)
-    * [Editing a client : `edit`](#editing-a-client--edit)
-    * [Locating clients by name : `find`](#locating-clients-by-name--find)
-    * [Deleting a client : `delete`](#deleting-a-client--delete)
+    * [Managing you clients](#managing-your-clients)
+      * [Listing all clients : `list`](#listing-all-clients--list) 
+      * [Adding a client : `add`](#adding-a-client--add)
+      * [Editing a client : `edit`](#editing-a-client--edit)
+      * [Deleting a client : `delete`](#deleting-a-client--delete)
+      * [Flagging a client : `flag`](#flagging-a-client--flag)
+      * [Unflagging a client : `unflag`](#unflagging-a-client--unflag)
+    * [Managing your meetings](#managing-your-meetings)
+      * [Scheduling / Rescheduling a meeting: `meet`](#scheduling--rescheduling-a-meeting-meet)
+      * [Canceling a meeting: `meet`](#canceling-a-meeting-meet)
+    * [Viewing your clients and meetings](#viewing-your-clients-and-meetings)
+      * [Sorting all clients : `sort`](#sorting-all-clients--sort)
+      * [Locating clients by name : `find`](#locating-clients-by-name--find)
+  * [**Useful Features**](#useful-features)
     * [Undoing the previous commands : `undo`](#undoing-the-previous-commands--undo)
     * [Redoing the previous commands : `redo`](#redoing-the-previous-commands--redo)
     * [Clearing all entries : `clear`](#clearing-all-entries--clear)
     * [Exiting the program : `exit`](#exiting-the-program--exit)
+  * [**Miscellaneous Features**](#miscellaneous-features)
     * [Saving the data](#saving-the-data)
     * [Editing the data file](#editing-the-data-file)
-  * [Command Summary](#command-summary)
+* [Command Summary](#command-summary)
   
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 For new users, it is highly recommended starting off from the `Quick Start` section to get HustleBook up and running.
 </div>
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Introduction
+Tired of how messy Excel sheets can get? HustleBook...
+
+--------------------------------------------------------------------------------------------------------------------
+
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have `Java 11` or above installed in your Computer. [What's Java 11?](https://www.oracle.com/java/)
 
-2. Download the latest `hustlebook.jar` from [here](https://github.com/AY2122S2-CS2103T-W15-2/tp/releases/).
+2. Download the latest `HustleBook` [here](https://github.com/AY2122S2-CS2103T-W15-2/tp/releases/).
+   ![Where to click](images/downloadHustleBookHere.png)
 
-3. Copy the file to the folder you want to use as the _home folder_ for your HustleBook.
+3. Place the file into a folder where you would like to keep it. We recommend keeping it in your `Desktop` for easier access.
```

Comment:
Sounds good, Maybe `save it and its data` instead of `keep it`? 

It would be good to add folder after `Desktop`, e.g. `Desktop` folder